editor.go

```diff
@@ -197,14 +197,11 @@ func (ed *Editor) rewriteLoadAndDeref(symbol string, derefOp uint8, length int,
 			return errors.Errorf("symbol %v: out of bounds dereference")
 		}
 
-		// Rewrite original load to new value
-		load.Constant = values[index]
```

Comment:
I think we should keep rewriting the load, otherwise cases where the load references rA and the deref rB don't work?

Reply:
In that case rB is the register that should actually hold the constant, rA is just a wasted register. AFAIK the only reason for clang to use rA again is for arrays. Perhaps we should check rA is not read from again?